examples/helper/wideresnet.py

```diff
@@ -0,0 +1,111 @@
+import math
```

Comment:
Is this model taken from somewhere or just standard? Just so people know about potential differences to a standard WRN.